docs/components/motor/_index.md

```diff
@@ -121,8 +68,16 @@ You can also ask questions on the [Viam Community Slack](https://join.slack.com/
 
 ## Next steps
 
+For general configuration and development info, see:
+
 {{< cards >}}
-{{% card link="/tutorials/get-started/confetti-bot/" %}}
-{{% card link="/tutorials/get-started/lazy-susan/" %}}
-{{% card link="/tutorials/configure/configure-rover" %}}
+{{% card link="/how-tos/configure/" noimage="true" %}}
+{{% card link="/how-tos/develop-app/" noimage="true" %}}
+{{% card link="/get-started/control-motor/" noimage="true" %}}
 {{< /cards >}}
+
+You can also use the motor component with the following services:
+
+- [motion service](/services/slam/): to move machines or components of machines
+- [navigation service](/services/navigation/): to navigate with GPS
+- [SLAM service](/services/slam/): for mapping
```

Comment:
```suggestion
```
See comment above; you'd use these services with a base or an arm but not with a motor AFAIK